test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -578,6 +578,7 @@ def load_config(self, cfg):
                              'file request received',
                              starttime=self.server.ctime)
         self.logger.info("Current config: %s" % cfg)
+        self.du.rm(hostname=self.serverA, path=fn, force=True)
```

Comment:
If manager() throws an exception, you'll leave this file lying around.  You might consider creating a class variable that is a list of files.  You can delete them in tearDown().